src/wallet.js

```diff
@@ -1746,6 +1771,21 @@ const wallet = {
     return this.decryptData(accessData.words, password);
   },
 
+  /**
+   * Get the privKey of the auth derivation
+   *
+   * @param {string} pin Pin to decrypt the words
```

Comment:
```suggestion
   * @param {string} pin Pin to decrypt the auth key
```

Reply:
Thanks! Done in 7be2bddfaa7280addf82144f2b3ef67767fac655